README.md

```diff
@@ -17,9 +19,7 @@ Martin & James will walk you through the fundamentals of analysis of the structu
 
 ### Run online
 
-Open in an interactive in-browser environment:
-
-[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/martinfleis/sdss22-workshop/HEAD?labpath=demo-notebook.ipynb)
+Open in an interactive in-browser environment: [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/martinfleis/sdss22-workshop/HEAD?labpath=demo-notebook.ipynb)
```

Comment:
What about these Binder buttons? I changed then to in-line. Think it's better to move them back?

Reply:
I like it more on a separate line as the buttons are not aligned with the text due to different sizes but that is just a personal preference and doesn't really matter.